extension/persistence/jpa-model/src/main/java/org/apache/polaris/jpa/models/ModelPolicyMappingRecord.java

```diff
@@ -33,6 +35,14 @@
           name = "POLICY_MAPPING_RECORDS_BY_POLICY_INDEX",
           columnList = "policyCatalogId,policyId,targetCatalogId,targetId")
     })
+@PrimaryKey(
```

Comment:
@dennishuo I think this is the same issue you observed previously, only applied to Policies this time right?